docs/source/prometheus.rst

```diff
@@ -26,6 +26,22 @@ dask_scheduler_clients
     Number of clients connected
 dask_scheduler_desired_workers
     Number of workers scheduler needs for task graph
+dask_scheduler_gil_contention
+    0-1 value representing GIL contention on scheduler
+
+    .. note::
+       Requires ``gilknocker`` to be installed, and 
+       ``distributed.admin.system-monitor.gil.enabled``
+       configuration to be set.
+
+dask_scheduler_gil_contention_total
+    Value representing cumulative total of GIL contention metrics since last scrape.
```

Comment:
```suggestion
    Value representing cumulative total of GIL contention.
```

IIUC, this value is never reset to zero, so it would be the grand total.

Reply:
What's the unit of measurement for this?

Reply:
You're right, that last bit ought to go. I'm not sure what the unit of measurement for this would be really when we're talking about 'total'. The metric only makes sense (a percentage) at a given moment of measurement.

Reply:
Reworded it in https://github.com/dask/distributed/pull/7651/commits/cb3a19e38d6d362d9bc36f23a0c6b1aa3585f90c, feel free to suggest further changes though. :)